### PR TITLE
feat(//converters): Add aten::embedding conversion functionality

### PR DESCRIPTION
**Pull request by peri044**
# Description

Support for aten::embedding conversion

Fixes https://github.com/NVIDIA/TRTorch/issues/102

## Type of change

Please delete options that are not relevant and/or add your own.

- New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: core/conversion/converters/impl/select.cpp [hunk 1]
```diff
@@ -117,6 +117,31 @@ auto select_registrations TRTORCH_UNUSED = RegisterNodeConversionPatterns()
 
             LOG_DEBUG("Output tensor shape: " << out->getDimensions());
 
+            return true;
+        }
+    }).pattern({
+        "aten::embedding(Tensor weight, Tensor indices, int padding_idx=-1, bool scale_grad_by_freq=False, bool sparse=False) -> (Tensor)",
+        [](ConversionCtx* ctx, const torch::jit::Node* n, args& args) -> bool {
+            auto embeddingWeight = args[0].unwrapToTensor();
```
Comment (narendasan) [inline core/conversion/converters/impl/select.cpp:125]: Is there a reason why we cant use ITensorOrFreeze here? This will create the const layer for you

Comment (peri044) [reply core/conversion/converters/impl/select.cpp]: Changed it now !!

**Comment by narendasan** [comment]
We merged a bunch of linting stuff can you rebase @peri044?

**Comment by peri044** [comment]
@narendasan  Done. Can you review and merge this ? Once this is done, I'll resolve slice layer PR to avoid conflicts.

**Pull request merged**